src/Quotient/Quotients.idr

```diff
@@ -0,0 +1,21 @@
+module Quotients
+
+import Quotient.Quotient
+
```

Comment:
Should this module use `%access public export`?

Reply:
Good point, forgot that.